dome9CloudBots/bots/storage_account_enable_https_traffic_only.py

```diff
@@ -0,0 +1,34 @@
+# What it does: enable Storage account Secure transfer required field
+# Usage: AUTO: storage_account_enable_https_traffic_only
+# Limitations: None
+
+import logging
+from msrestazure.azure_exceptions import CloudError
+from azure.mgmt.storage import StorageManagementClient
+from azure.mgmt.storage.models import StorageAccountUpdateParameters
+
+
+def run_action(credentials, rule, entity, params):
+    logging.info(f'{__file__} - run_action started')
```

Comment:
Could be replaced with 
`logging.info(f'{__file__} - {foo.__name__} started')`

If you rename this function, at least you won't forget to change it in the logger as well.